docs/getting-started/advanced-setting.asciidoc

```diff
@@ -187,15 +187,17 @@ The `securitySolution:maxUnassociatedNotes` field determines the maximum number
 
 [discrete]
 [[exclude-cold-frozen-data-rule-executions]]
-== Exclude cold and frozen data from rule executions 
+== Exclude cold and frozen data from rules 
 
-To ensure rules don't search cold and frozen data when executing, specify cold and frozen {ref}/data-tiers.html[data tiers] in the `excludedDataTiersForRuleExecution` field. Multiple data tiers must be separated by commas, for example: `data_frozen`, `data_cold`. This setting is turned off by default; turning it on can improve rule performance and reduce execution time. 
+To ensure the rules in your {kib} space exclude query results from cold and frozen tiers when executing, specify cold and frozen {ref}/data-tiers.html[data tiers] in the `excludedDataTiersForRuleExecution` field. Multiple data tiers must be separated by commas, for example: `data_frozen`, `data_cold`. This setting is turned off by default; turning it on can improve rule performance and reduce execution time. 
 
 This setting does not apply to {ml} rules. 
```

Comment:
During my convo with @MikePaquette yesterday, he mentioned that ML results are stored in a system index that can't be modified by users. Maybe that detail is worth mentioning too? For example:

```suggestion
This setting does not apply to {ml} rules because {ml} anomalies are stored in a system index that is never frozen. 
```

Reply:
It's not a system index. Suggest `This setting does not apply to {ml} rules because {ml} anomalies are not stored in cold or frozen.`

Reply:
Minor adjustment for specificity:

```suggestion
This setting does not apply to {ml} rules because {ml} anomalies are not stored in cold or frozen data tiers.
```